app/models/manageiq/providers/hawkular/datawarehouse_manager.rb

```diff
@@ -46,10 +46,17 @@ def self.raw_connect(hostname, port, token, alerts = false)
     end
 
     def connect(options = {})
-      @client ||= self.class.raw_connect(hostname,
-                                         port,
-                                         authentication_token('default'),
-                                         options[:alerts])
+      @clients ||= {}
```

Comment:
I think they use the returned variable and only use `@client` and the new `@clients` to save the connection for future calls.
Is that right @moolitayer  ?